src/app/exam/score/ExamScorePage.js

```diff
@@ -33,30 +31,45 @@ const ExamScorePage = () => {
         return <Spinner/>;
     }
 
+    const onButtonExportCSVClick = (examId) => {
+        examService.getExamTranscriptCsvFile(examId)
+        .then((res) => {
+            const url = window.URL.createObjectURL(new Blob([res.data]));
+            const link = document.createElement('a');
+            link.href = url;
+            link.setAttribute('download', 'ExamTranscript.csv');
+            link.click();
+        });
+    };
+
     return (
         <div className="exam-score">
-            <div className="font-poppins" style={{paddingTop: "20px", paddingBottom: "150px"}}>
-                <div className="mt-2" style={{display: "flex", justifyContent: "center"}}>
-                    <div className="mt-4 mr-3" style={{width: "fit-content", textAlign: "center"}}>
+            <div className="exam-score-font-poppins">
+                <div className="exam-score-mt-2">
+                    <div className="exam-score-mr-3">
```

Comment:
`mt-2`, `mr-3`, `font-poppins` are the syntactic sugar provided by Bulma.
For example, `mt-2` means `margin-top: 0.5rem`.

That is, if the className is `exam-score-mt-2` then `mt-2` is out of action.